libafl/src/events/tcp.rs

```diff
@@ -706,16 +631,12 @@ where
         + HasCurrentTestcase<I>
         + Stoppable,
     I: DeserializeOwned,
-    Z: ExecutionProcessor<Self, I, E::Observers, S> + EvaluatorObservers<E, Self, I, S>,
 {
-    fn process(&mut self, fuzzer: &mut Z, state: &mut S, executor: &mut E) -> Result<usize, Error> {
+    fn receive(&mut self, state: &mut S) -> Result<Option<(Event<I>, bool)>, Error> {
```

Comment:
`receive_available`?
or `try_receive`?
Receive sounds like it would be blocking